presto-main/src/main/java/com/facebook/presto/sql/planner/PlanOptimizers.java

```diff
@@ -409,6 +412,19 @@ public PlanOptimizers(
                         estimatedExchangesCostCalculator,
                         ImmutableSet.of(new RemoveTrivialFilters())),
                 new UnaliasSymbolReferences(metadata.getFunctionAndTypeManager()),
+                new ReplaceConstantVariableReferencesWithConstants(metadata.getFunctionAndTypeManager()),
+                simplifyRowExpressionOptimizer,
+                new ReplaceConstantVariableReferencesWithConstants(metadata.getFunctionAndTypeManager()),
```

Comment:
Lets run `PruneUnreferencedOutputs` after the second `ReplaceConstantVariableReferencesWithConstants` ?

Reply:
Added